packages/product-catalogue/package.json

```diff
@@ -32,11 +32,16 @@
   "bugs": {
     "url": "https://github.com/opensrp/web/issues"
   },
-  "devDependencies": {
-    "@opensrp/store": "^0.0.3"
-  },
   "dependencies": {
-    "@opensrp/reducer-factory": "^0.0.7"
+    "@onaio/redux-reducer-registry": "^0.0.9",
+    "@opensrp/notifications": "^0.0.1",
+    "@opensrp/react-utils": "^0.0.1",
+    "@opensrp/reducer-factory": "^0.0.7",
+    "@opensrp/server-service": "^0.0.11",
+    "@opensrp/store": "^0.0.6",
+    "antd": "^4.8.2",
+    "formik-antd": "2.0.1",
+    "yup": "^0.29.3"
```

Comment:
is it possible to use antd validations instead of Yup?  And should these be peer dependencies?

Reply:
moved antd to peer. removed the other 2 since they are not being used in ths pr. 
However my choice to use yup was because: 
I went with yup, beacuse i am more familiar with it, we've used it extensively elsewhere. formik itself recommends it. 
Is there a reason why we cannot use the two?

Reply:
@p-netm it seems there's a lot of debate about what to use for the form.  I think we should pick something that everyone agrees with and then use it across the board.  My only preference is that we use formik.